packages/owl-bot-post-processor/src/bin/commands/trigger-build.ts

```diff
@@ -11,10 +11,30 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+import {
+  createCheck,
+  getAuthenticatedOctokit,
+  getGitHubShortLivedAccessToken,
+  getOwlBotLock,
+  triggerBuild,
+} from '../../core';
+import {promisify} from 'util';
+import {readFile} from 'fs';
 import yargs = require('yargs');
-import {BuildArgs, createCheck, triggerBuild} from '../../core';
 
-export const triggerBuildCommand: yargs.CommandModule<{}, BuildArgs> = {
+const readFileAsync = promisify(readFile);
+
+interface Args {
+  'pem-path': string;
+  'app-id': number;
+  installation: number;
+  repo: string;
```

Comment:
Decided while working on this PR that @JustinBeckwith was right, I'd rather have the arguments be `appId`, `pemPath`, once passed to the core utilities.